src/app/lib/config/services/ukrainian.js

```diff
@@ -5,275 +5,410 @@ import '@bbc/moment-timezone-include/tz/GMT';
 import '@bbc/psammead-locales/moment/uk';
 import withContext from '../../../contexts/utils/withContext';
 
-export const service = {
-  default: {
-    ads: {
-      hasAds: false,
-      advertisementLabel: 'Реклама',
-    },
-    lang: `uk`,
-    articleAuthor: `http://www.facebook.com/bbcnews`,
-    articleTimestampPrefix: 'Оновлення даних: ',
-    atiAnalyticsAppName: 'news-ukrainian',
-    atiAnalyticsProducerId: '94',
-    chartbeatDomain: 'ukrainian.bbc.co.uk',
-    brandName: 'BBC News Україна',
-    product: 'BBC News',
-    serviceLocalizedName: 'Україна',
-    defaultImage: 'https://news.files.bbci.co.uk/ws/img/logos/og/ukrainian.png',
-    defaultImageAltText: 'BBC News Україна',
-    dir: `ltr`,
-    externalLinkText: ', зовнішнє посилання',
-    imageCaptionOffscreenText: 'Підпис до фото, ',
-    videoCaptionOffscreenText: 'Підпис до відео, ',
-    audioCaptionOffscreenText: 'Підпис до аудіо',
-    defaultCaptionOffscreenText: 'Підпис, ',
-    imageCopyrightOffscreenText: 'Автор фото, ',
-    locale: `uk-UA`,
-    datetimeLocale: `uk`,
-    service: 'ukrainian',
-    serviceName: 'Ukrainian',
-    themeColor: `${C_POSTBOX}`,
-    twitterCreator: '@BBC_ua',
-    twitterSite: '@BBC_ua',
-    noBylinesPolicy:
-      'https://www.bbc.com/ukrainian/institutional-50170368#authorexpertise',
-    publishingPrinciples:
-      'https://www.bbc.com/ukrainian/institutional-50170368',
-    isTrustProjectParticipant: true,
-    script: cyrillicAndLatin,
-    manifestPath: '/manifest.json',
-    swPath: '/sw.js',
-    frontPageTitle: 'Новини',
-    theming: {
-      brandBackgroundColour: `${C_POSTBOX}`,
-      brandLogoColour: `${C_WHITE}`,
+const baseContext = {
```

Comment:
Following the services with variants:
```suggestion
const baseServiceConfig = {
```